src/features/creators/creationHelpers.ts

```diff
@@ -0,0 +1,149 @@
+import { window, extensions, Uri } from 'vscode';
+import * as fs from 'fs-extra';
+import * as path from 'path';
+import { EnvironmentManagers } from '../../internal.api';
+import { CreateEnvironmentOptions, EnvironmentManager } from '../../api';
+import { traceVerbose } from '../../common/logging';
+
+/**
+ * Prompts the user to choose whether to create a new virtual environment (venv) for a package.
+ * @returns {Promise<boolean | undefined>} Resolves to true if 'Yes' is selected, false if 'No', or undefined if cancelled.
+ */
+export async function promptForVenv(): Promise<boolean | undefined> {
+    const venvChoice = await window.showQuickPick(['Yes', 'No'], {
+        placeHolder: 'Would you like to create a new venv for this package?',
```

Comment:
We should probably say "virtual environment" in case the user has a different default that `venv`
```suggestion
        placeHolder: 'Would you like to create a new virtual environment for this package?',
```